### DISCUSSION TITLE
Autologin with SSO-provider

### DISCUSSION THREAD

**Discussion by jakobwarnhjelm** [Budibase/budibase#13170] [Help]
Is it possible to configure Budibase so that the user doesn't have to press the SSO button, instead it is redirected automatically?

**Comment by ConorWebb96** [comment]
Hey @jakobwarnhjelm,

The only way this will be possible is if a change to the core product was made. Currently you can't, however you can force your users to only be able to use SSO with [enforced sso](https://docs.budibase.com/docs/enforced-sso). This disables password accounts and only allows access to the platform via SSO.

If you think this would be a popular change, I'd recommend creating a feature request on Github and the team will take it from there. Whether or not this gets accepted is completely up to the product team and it will highly be dependant on how popular of a request it is.
